### PR TITLE
workflow: show latest release in resolve action

### PR DESCRIPTION
**Pull request by dcramer**
#nochanges

### CODE REVIEW

**Comment by dcramer** [comment]
Not sure if this works on mysql (waiting on CI)

**Comment by dcramer** [comment]
Before landing this we need to confirm performance on large accounts

**Comment by dcramer** [comment]
This includes a pretty big refactor of the project details code and serialization to avoid the query cost this would have added.

File: src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx
Comment (dcramer) [inline src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx]: we could probably remove all of this review code at this point

**Comment by dcramer** [comment]
@mitsuhiko killed some of the callsign code here

File: src/sentry/api/serializers/models/project.py
Comment (mitsuhiko) [inline src/sentry/api/serializers/models/project.py]: not sure if this can happen, but if item_list is empty the sql query will have a syntax error.

Comment (dcramer) [inline src/sentry/api/serializers/models/project.py]: i confirmed serializers get skipped if the item_list (or whatever) is falsish

**Review by mattrobenolt — APPROVED** [review]
wcgw

**Review by mitsuhiko — APPROVED** [review]

**Pull request merged**
